ground_truth_labeling_jobs/multi_modal_parallel_sagemaker_labeling_workflows_with_step_functions/sagemaker_ground_truth_workflows.ipynb

```diff
@@ -0,0 +1,817 @@
+{
```

Comment:
<p>Please add a space between 'instance.' and 'if' in the first bullet (second sentence).</p><p> </p><p>Remove "please" - this is not AWS style.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>